site/gatsby-site/src/components/users/UsersField.js

```diff
@@ -31,11 +31,16 @@ export default function UsersField({ id, name, placeHolder = '' }) {
   useEffect(() => {
     if (data?.users) {
       setSelected((selected) =>
-        selected.map(({ id }) => {
-          const { userId, first_name, last_name } = data.users.find((user) => user.userId == id);
+        selected
+          .map(({ id }) => {
+            const user = data.users.find((user) => user.userId == id);
 
-          return { id: userId, first_name, last_name };
-        })
+            if (!user) return null;
```

Comment:
This will map `null` objects in the array. I think it should filter then instead.